torch/_inductor/codegen/triton.py

```diff
@@ -1254,6 +1254,8 @@ def __init__(
         self.iter_vars_count = itertools.count()
         self.inside_reduction = self.numels[-1] != 1
         self.body = IndentedBuffer()
+        self.prologue_A_body = IndentedBuffer()
+        self.prologue_B_body = IndentedBuffer()
```

Comment:
Can this be done without special-casing the fact that there are two input named A and B?

Some templates might take more (or less) args and name them differently.